Shared/Realm.Shared/Linq/IRealmCollection.cs

```diff
@@ -113,6 +113,7 @@ public interface IRealmCollection<out T> : IReadOnlyList<T>, INotifyCollectionCh
         /// Gets the <see cref="ObjectSchema"/> of the contained objects.
         /// </summary>
         /// <value>The ObjectSchema of the contained objects.</value>
+        [Obsolete("This property is no longer necessary and will be removed in a subsequent release.")]
```

Comment:
Just because we don't use it internally anymore doesn't mean it lacks intrinsic value in and of itself. Please leave as it was.